com.warlordsoftwares.formatlab.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.warlordsoftwares.formatlab
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-23.08'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+command: formatlab.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfsd
+  - --device=dri
+  - --socket=pulseaudio
+tags:
+  - proprietary
+cleanup:
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: libass
+    sources:
+      - type: archive
+        url: https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.xz
+        sha256: f0da0bbfba476c16ae3e1cfd862256d30915911f7abaa1b16ce62ee653192784
+      - type: script
+        commands:
+          - autoreconf -fiv
+        dest-filename: autogen.sh
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share
+      - '*.a'
+      - '*.la'
+  - name: luajit
+    no-autogen: true
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: git
+        url: https://github.com/LuaJIT/LuaJIT.git
+        commit: 4c2441c16ce3c4e312aaefecc6d40c4fe21de97c
+  - ffmpeg.json
+  - mpv.json
+  - python3-req.json
+  - name: formatlab
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/formatlab
+      - tar -xzf *latest.tar.gz -C ${FLATPAK_DEST}/formatlab
+      - install -Dm755 ${FLATPAK_DEST}/formatlab/FLATPAK_FORMAT_LAB-latest/formatlab.sh ${FLATPAK_DEST}/bin/formatlab.sh
+      - install -Dm644 ${FLATPAK_DEST}/formatlab/FLATPAK_FORMAT_LAB-latest/share/icons/icon.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/com.warlordsoftwares.formatlab.png
+      - install -Dm644 ${FLATPAK_DEST}/formatlab/FLATPAK_FORMAT_LAB-latest/share/applications/formatlab.desktop ${FLATPAK_DEST}/share/applications/com.warlordsoftwares.formatlab.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/com.warlordsoftwares.formatlab.desktop
+      - install -Dm644 ${FLATPAK_DEST}/formatlab/FLATPAK_FORMAT_LAB-latest/share/metainfo/com.warlordsoftwares.formatlab.metainfo.xml ${FLATPAK_DEST}/share/metainfo/com.warlordsoftwares.formatlab.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/rishabh3354/FLATPAK_FORMAT_LAB/archive/refs/tags/latest.tar.gz
```

Comment:
this should be a specific commit or tag. `latest` mean that next time it's update the build will break, and no one will be able to reproduce the previous built.

Reply:
Got it, It is now resolved!